monitoring/local_monitoring/local_monitoring_demo.sh

```diff
@@ -149,22 +149,43 @@ function kruize_local() {
 ###########################################
 function get_urls() {
 	kubectl_cmd="kubectl -n default"
-	TECHEMPOWER_PORT=$(${kubectl_cmd} get svc tfb-qrh-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
-	TECHEMPOWER_IP=$(${kubectl_cmd} get pods -l=app=tfb-qrh-deployment -o wide -o=custom-columns=NODE:.spec.nodeName --no-headers)
-
-	if [ ${CLUSTER_TYPE} == "minikube" ]; then
+	if [ ${CLUSTER_TYPE} == "kind" ]; then
 		kubectl_cmd="kubectl -n monitoring"
 
-		MINIKUBE_IP=$(minikube ip)
-
-		KRUIZE_PORT=$(${kubectl_cmd} get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
-		KRUIZE_UI_PORT=$(${kubectl_cmd} get svc kruize-ui-nginx-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+		KIND_IP=127.0.0.1
```

Comment:
KindIP should be fetch like this
Kind_IP=$(kubectl get nodes -o wide --no-headers | awk '{print $6}')